conpot/protocols/http/command_responder.py

```diff
@@ -799,9 +802,13 @@ def do_GET(self):
             get_data = self.rfile.read(int(get_data_length))
 
         # try to find a configuration item for this GET request
-        entity_xml = configuration.xpath(
-            '//http/htdocs/node[@name="' + self.path.partition('?')[0] + '"]'
-        )
+        try:
+            entity_xml = configuration.xpath(
+                '//http/htdocs/node[@name="' + self.path.partition('?')[0] + '"]'
+            )
+        except:
```

Comment:
Maybe catching specific exceptions is better. As per #399 I think we can catch `lxml.etree.XPathEvalError` here.
```python
from lxml.etree import XPathEvalError
...
try:
    # make unicorns
except XPathEvalError as xe:
    logger.debug('Uh oh! Malformed HTTP ... : {}.format(xe))
    # make unicorns after catching exceptions
```

Reply:
That's why I referred to it as "not very PEP-8" ;-) .. but you're right of course.